ocs_ci/helpers/helpers.py

```diff
@@ -835,6 +835,30 @@ def validate_cephfilesystem(fs_name):
     return True if (ceph_validate and ocp_validate) else False
 
 
+def delete_all_resource_of_kind_containing_string(kind, search_string):
```

Comment:
this function could be quite risky in case a certain string is passed while, and this string is also part of the name of other unrelated cluster resrouces. 
Is there any other way to delete the needed resources that is more explicit?

Reply:
I saw now an implementation that teardown can use an array that can be appended in the middle of the test. This could be helpful and more explicit. I'll give it a shoot.

Reply:
Did a different implementation to teardown without using this function. Should I delete it from helpers?

Reply:
You already created a more explicit function, we can probably get rid of this one.

Reply:
@ebenahar @am-agrawa What do you  think of making the string constant to "test".

Reply:
discussed offline and since this function will not be in use, it will be removed